examples/sagemaker-tensorflow/sagemaker-merlin-tensorflow.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>how <code>train.py</code> is uploaded to the container? does user need to mount a volume  to container?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/692/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I added a couple of sentences to clarify. In short, when we simply specify the local path to the file here, the sagemaker python sdk will take care of uploading it to S3, and downloading it to the proper path when the training instance is launched.